src/send.js

```diff
@@ -70,6 +70,9 @@ const send = (
           )
         );
       } catch (error) {
+        if (process.env.NODE_ENV === 'development') {
```

Comment:
We're not using webpack, so I'm not sure babel strips the process global variable. This might produce a ReferenceError. I have to test before merging.